lib/improver/tests/cli/test_init.py

```diff
@@ -37,7 +37,8 @@
 from improver.cli import (
     docutilize, unbracket,
     maybe_coerce_with, inputcube, inputjson, with_output,
-    with_intermediate_output)
+    with_intermediate_output, create_constrained_inputcube_converter)
+from improver.utilities.load import load_cube
```

Comment:
I don't think we should be loading any cubes here - so shouldn't be importing

Reply:
In the mocking, it is testing that load_cube is called with the constraint.
```python
        m.assert_any_call(load_cube, "foo", constraints='wind_speed')
        m.assert_any_call(load_cube, "foo", constraints='wind_from_direction')
```

Reply:
Ah, my bad - if it's being mocked then I'm happy.